### ISSUE TITLE
Stuck at-- Step 2: Use an external package

### ISSUE THREAD

**Issue by Enigmanomalous** [flutter/flutter#15063]
From URL: https://flutter.io/get-started/codelab/index.md
       
Just stumbling through the tutorial . . . LOL
### Stuck at-- Step 2: Use an external package
         

![screenshot from 2018-03-02 05-34-57](https://user-images.githubusercontent.com/36937018/36899319-c24908fe-1ddb-11e8-91ad-b636edb514b2.png)

The only thing the editor suggests while typing "word" is generate**Word**Pairs()

This tells me that the package has not loaded/imported correctly . . .
So I attempted to reload it and its dependencies as follows:

![english_words packages get](https://user-images.githubusercontent.com/36937018/36900109-256c607c-1ddf-11e8-9047-e3a9d604a44e.png)

![english_words packages get result](https://user-images.githubusercontent.com/36937018/36900115-2e1b0d40-1ddf-11e8-9a89-76b6084919bf.png)

![english_words get dependencies result](https://user-images.githubusercontent.com/36937018/36900118-328bafce-1ddf-11e8-8a39-4a8f75227dfb.png)

Still no change, and with all that red ink I was pretty sure that something is not working with the dependencies get . . .
So I attempted the the only appropriate Terminal operation that I could find ...

![english_words terminal result](https://user-images.githubusercontent.com/36937018/36900242-c1d491d2-1ddf-11e8-9835-44d24bd9be36.png)

As you can see this executes just fine but has no effect on the dependencies issue.

I would like to think that I am bright enough to follow the simple instructions, but apparently I'm missing something:

- No idea why it would complain-- "The Flutter SDK is not available."

- No idea how to look up what "Process finished with exit code 69" means, but I'm guessing that it has something to do with "Flutter SDK is not available", which makes no sense at all because it is able to complete the Packages Get operation just fine.

So, what am I missing? ? ?

**Comment by mostafabahri** [comment]
What's the `flutter doctor` output?
could be https://github.com/flutter/flutter/issues/10671

**Comment by Enigmanomalous** [comment]
timberlane@TMBR ~/AndroidStudioProjects/startup_namer $ flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, v0.1.5, on Linux, locale en_US.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
[✓] Android Studio (version 3.0)
[✓] Connected devices (1 available)

• No issues found!

**Comment by mostafabahri** [comment]
Try running the command inside Android Studio :  **Tools > Flutter > Flutter Doctor** .
Have you tried launching Android Studio from a terminal?

**Comment by Enigmanomalous** [comment]
@mostafabahri Try running the command inside Android Studio :
The result
/home/timberlane/flutter/bin/flutter --no-color doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, v0.1.5, on Linux, locale en_US.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
[✓] Android Studio (version 3.0)
[✓] Connected devices (1 available)

• No issues found!
Process finished with exit code 0

**Comment by Enigmanomalous** [comment]
@mostafabahri Have you tried launching Android Studio from a terminal?

Same result - - No cahnge

**Comment by mit-mit** [comment]
Hi,

The output from https://user-images.githubusercontent.com/36937018/36900115-2e1b0d40-1ddf-11e8-9a89-76b6084919bf.png suggests that the 'packages get' command did complete as expected.

But I am confused that you are also seeing https://user-images.githubusercontent.com/36937018/36900118-328bafce-1ddf-11e8-8a39-4a8f75227dfb.png -- what are you clicking in the IDE to see that?

cc @devoncarew

**Comment by mit-mit** [comment]
Got it, that whole line of actions is not expected for Flutter apps. If you use the get action in the Flutter line, can you move forward in getting started?

**Comment by Enigmanomalous** [comment]
@mit-mit use the get action in the Flutter line
I am confused by this - - get action? What is the Flutter line?

**Comment by mostafabahri** [comment]
@Enigmanomalous the line that says Flutter commands. click `Packges get`.
I clicked `get depenendices` in the IDE and got the same error as you.

**Comment by mostafabahri** [comment]
@mit-mit is there any use case for using `pub get` in a Flutter project? 
it's just confusing to be there if there is no use case.

**Comment by mit-mit** [comment]
There is not. It's a bug for that first line to be shown.

**Comment by Enigmanomalous** [comment]
Ok. So the get dependencies error is a Red-herring. Understood.

Still no explanation why _wordPair.asPascalCase_ is not recognized in the code editor (_Undefined name 'wordPair'_) and throughs a recursive error when attempting to execute it.

**Comment by mit-mit** [comment]
I think you are missing this line?

`final wordPair = new WordPair.random();` ?

**Comment by Enigmanomalous** [comment]
LOL
Bonanza !!!
I did so totally miss that line change to the tutorial code.
I got focused on the part that had **_//notes_** around it . . .
Perhaps we could add a note to draw attention to that instantiation code?

I ran the app and it generated '_WildFaith_' . . .  How awesome is that because that is where I'm at, jumping into Flutter with a huge commitment for a mission critical dev project for a startup company. No pressure in that. LOL  WildFaith sounds like a great codename for the first build let out into the wild!

I have been away from coding for over a decade and am new to the language, the IDE, mobile app dev., and of course to Flutter. 
Thanks mit-mit sometimes all we need is a fresh set of eyes.
